euphoria-flink/src/test/java/cz/seznam/euphoria/flink/testkit/FlinkExecutorProvider.java

```diff
@@ -32,6 +33,9 @@ default ExecutorEnvironment newExecutorEnvironment() throws Exception {
     RocksDBStateBackend backend = new RocksDBStateBackend("file://" + path);
     FlinkExecutor executor = new TestFlinkExecutor(ModuloInputSplitAssigner::new)
         .setStateBackend(backend);
+    // TODO should use some local test implementation of counters
+    // that is queryable in assertions
```

Comment:
Is this comment meant for the sake of unittesting user flows? Isn't this what `InMemExecutor` is for?